coralproject/talk
# [CORL-349] Show banned user info

## What does this PR do?
- When a banned user is logged in, the ui will show a message and relfects the banned status my disabled or removing buttons that the user can no longer do

## How do I test this PR?

- Ban user
- Login as user
- See banned user message
- See buttons removed or disabled
